FaustCode/owl.cpp

```diff
@@ -446,7 +661,10 @@ class OwlUI : public UI {
         : fPatch(pp)
         , fParameter(NO_PARAMETER)
         , fParameterIndex(0)
+        , fSoundfileIndex(0)
         , fButton(NO_BUTTON) {
+        fSoundReader = new OwlResourceReader();
```

Comment:
shouldn't this be NULL here and instantiated on line 737?

Reply:
Correct, removed this and tested that it still works.